### PR TITLE
docs: fix dolphin demo

### PR DESCRIPTION
**Pull request by vagetablechicken**

### CODE REVIEW

**Comment by vagetablechicken** [comment]
DolphinScheduler release should contain https://github.com/apache/dolphinscheduler/commit/43e6ac32ddc468c57633162a5cd442a70f418c18, 3.1.2 has bug in OpenMLDB Task

**Review by tobegit3hub — APPROVED** [review]
LGTM

File: docs/zh/use_case/dolphinscheduler_task_demo.md [hunk 1]
```diff
@@ -48,50 +48,61 @@ dpkg-reconfigure dash
...
 ```
-python3 /work/talkingdata/predict_server.py --no-init > predict.log 2>&1 &
+```{tip}
+如果在“在线预测测试”中返回了错误，请查看日志`/work/predict.log`。
 ```
 
-**运行 DolphinScheduler**
+**下载并运行 DolphinScheduler**
+
+DolphinScheduler 支持 OpenMLDB Task 的版本，我们直接提供了一个可供下载版本，下载地址[dolphinscheduler-bin](http://openmldb.ai/download/dolphinschduler-task/apache-dolphinscheduler-dev-SNAPSHOT-bin.tar.gz)。
```
Comment (lumianph) [inline docs/zh/use_case/dolphinscheduler_task_demo.md:77]: does it has an official release?

Comment (vagetablechicken) [reply docs/zh/use_case/dolphinscheduler_task_demo.md:77]: It's better to wait 3.1.3, the current version 3.1.2 can't run OpenMLDB Task

**Review by dl239 — APPROVED** [review]

**Pull request merged**